tests/integrations_app/flagship/test_flashy.py

```diff
@@ -0,0 +1,13 @@
+import os
+
+import pytest
+from integrations_app.flagship import _PATH_INTEGRATIONS_DIR
+
+from lightning_app.testing.testing import run_app_in_cloud
+
+
+@pytest.mark.cloud
+def test_app_cloud() -> None:
+    with run_app_in_cloud(os.path.join(_PATH_INTEGRATIONS_DIR, "flashy")) as (_, _, fetch_logs, _):
+
+        print(fetch_logs)  # TODO
```

Comment:
What is this test meant to do? 

Note that app smoke tests are evil at the moment: They rely on the cloud to be working. If it isn't, it blocks all development, ALL.

Reply:
yes, I would like to ask @ethanwharris to check what we shall be validation to know that App is running correctly :otter:

Reply:
For Flashy you should be able to adapt the test from here: https://github.com/Lightning-AI/LAI-Flashy-App/blob/f58d772601542213c50b7e4f81e973931ca1e4d9/tests/test_app_gallery.py#L205